### PR TITLE
Add automatic initialization of DB schema.

### PR DESCRIPTION
**Pull request by Artanicus**
As long as the DB exists and we have permissions to create tables it
should work.

### CODE REVIEW

File: core/bot/db.php [hunk 1]
```diff
@@ -1,13 +1,32 @@
 <?php
+
+require_once('logic/sql_utils.php');
```
Comment (Ninjasoturi) [inline core/bot/db.php:3]: All the logic stuff is included in `core/bot/requirements.php` that in turn is included before `core/bot/db.php` in index.php, so this is not needed.

File: docs/manual_install.rst [hunk 2]
```diff
@@ -40,26 +40,19 @@ Bot Settings:
...
 --------
 
-Create a new mysql database and user for your bot.
+Create a new mysql database and user for your bot, granting full priviledges for the db. For example:
 
-Command to access the terminal as root user: ``mysql -u root -p``
+.. code-block:: shell
+    mysql -u root -p
+    MariaDB [(none)]> CREATE USER 'username'@'localhost' IDENTIFIED BY 'PASSWORD';
+    MariaDB [(none)]> CREATE DATABASE databasename;
```
Comment (Ninjasoturi) [inline docs/manual_install.rst:48]: `MariaDB [(none)]>` do we want these here?

File: docs/manual_install.rst [hunk 3]
```diff
@@ -40,26 +40,19 @@ Bot Settings:
...
+    MariaDB [(none)]> GRANT ALL PRIVILEGES ON databasename.* TO 'username'@'localhost';
+    MariaDB [(none)]> FLUSH PRIVILEGES;
+    MariaDB [(none)]> exit
 
-Command to create a user with localhost access (Only allow localhost access!): ``CREATE USER 'USERNAME'@'localhost' IDENTIFIED BY 'PASSWORD';``
+The DB structure is imported the first time the bot is used. If it fails for any reason you can also run it manually:
```
Comment (Ninjasoturi) [inline docs/manual_install.rst:53]: Should this be also menationed in docker side of documentation?

**Review by Ninjasoturi — COMMENTED** [review]
Tested it and it worked. I'm fine with merging this.

File: docs/manual_install.rst (hunk 2 above)
Comment (Artanicus) [reply docs/manual_install.rst:48]: ACK

Comment (Artanicus) [reply docs/manual_install.rst:48]: Good point, simplified this quite a bit.

File: docs/manual_install.rst (hunk 3 above)
Comment (Artanicus) [reply docs/manual_install.rst:53]: Done

**Pull request merged**